### DISCUSSION TITLE
SSL_connect returned=1 errno=0 state=error: unsupported protocol

### DISCUSSION THREAD

**Discussion by moderatp** [postalserver/postal#2078] [Help with using Postal]
Hello everyone and I ask for help.
The problem with sending a message, I don’t even know what to do anymore, the log worker is below.
worker_1    | [9] [2022-07-20T08:34:33.362] INFO -- : [IYLYQEIX] Cannot connect to 46.XXX.X40.XX:25 (smtp.XXX.COM) (**OpenSSL::SSL::SSLError: SSL_connect returned=1 errno=0 state=error: unsupported protocol**)
worker_1    | [9] [2022-07-20T08:34:33.364] INFO -- : [IYLYQEIX] -> No SMTP server available for pwrs.ru
worker_1    | [9] [2022-07-20T08:34:33.364] INFO -- : [IYLYQEIX] -> Hostnames: ["smtp.XXX.COM"]
worker_1    | [9] [2022-07-20T08:34:33.364] INFO -- : [IYLYQEIX] -> **Errors: ["SSL_connect returned=1 errno=0 state=error: unsupported protocol"]**
worker_1    | [9] [2022-07-20T08:34:33.381] INFO -- : [IYLYQEIX] Finishing up

Configuration of the main settings, also attached below.
general:
  use_ip_pools: false

web:
  host: corp.ХХХ.com
  protocol: https

web_server:
  bind_address: 127.0.0.1
  port: 5000

smtp_server:
  port: 25
  tls_enabled: true
  tls_ciphers: AES256-SHA
  ssl_version: TLSv1 #SSLv1
  tls_certificate_path: /config/smtp.crt
  tls_private_key_path: /config/smtp.key
  log_connect: true
  proxy_protocol: false
  strip_received_headers: false
  max_message_size: 32 # size in Megabytes

logging:
  stdout: true

There is the latest version of Postal 2.1.1 installed in the standard way

**Comment by willpower232** [comment]
Have a look at #982

**Reply by moderatp** [reply]
[#982 ](https://github.com/postalserver/postal/issues/982) no solution to my problem =(

**Reply by willpower232** [reply]
it sounds like you either need to lower the OpenSSL config like this https://github.com/postalserver/postal/issues/982#issuecomment-747620856 or convince whatever you're connecting to to upgrade.

**Reply by willpower232** [reply]
see also https://github.com/postalserver/postal/discussions/1558
